kieker-documentation/userguide/bibliography.bib

```diff
@@ -1,5 +1,5 @@
 % Replaced by correct values during build process
```

Comment:
This is updated automatically, so it would not be necessary to update it here if nothing else changed.